server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/tservers.ftl

```diff
@@ -49,6 +49,7 @@
             </thead>
             <tbody></tbody>
           </table>
+          <span id="recovery-caption" style="background-color: gold;"></span>
```

Comment:
Instead of using a custom inline style, would it be possible to use the built-in style from bootstrap? https://getbootstrap.com/docs/4.0/utilities/colors/#background-color

Reply:
I can't get this working for some reason. I was able to apply the class to the caption, but the datatable will not show the change and I'm not sure why.

Reply:
:shrug: